### PR TITLE
wmp11: add new verb

### PR DESCRIPTION
**Pull request by soredake**
Currently tested with Resident Evil 5/6 in 64-bit prefix with proton.

### CODE REVIEW

**Comment by mmbossoni** [comment]
I've tested with Revelations 2. Seems to be working correctly.

File: src/winetricks [hunk 1]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+    title="Windows Media Player 11" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="wmp11-windowsxp-x64-enu.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Windows Media Player/wmplayer.exe"
+
+load_wmp11()
+{
+    #w_package_unsupported_win64
```
Comment (austin987) [inline src/winetricks]: You probably want w_package_unsupported_win32 instead (or does it support 32-bit prefixes).

File: src/winetricks [hunk 2]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+
+    #w_call gdiplus
+
+    # http://down.cdhaha.net/?p=39
+    w_download https://web.archive.org/web/20190512112704/https://download.microsoft.com/download/3/0/8/3080C52C-2517-43DE-BDB4-B7EAFD88F084/wmp11-windowsxp-x64-enu.exe 5af407cf336849aff435044ec28f066dd523bbdc22d1ce7aaddb5263084f5526
+
+    w_set_winver winxp
+
+    # remove builtin placeholders to allow update
+    rm -f "$W_SYSTEM64_DLLS"/wmvcore.dll "$W_SYSTEM64_DLLS"/wmp.dll
```
Comment (austin987) [inline src/winetricks]: Prefix with `w_try` (and preferably sort the entries).

File: src/winetricks [hunk 3]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+    rm -f "$W_PROGRAMS_UNIX/Windows Media Player/wmplayer.exe"
```
Comment (austin987) [inline src/winetricks]: Same (could be combined with line above, but separate is okay too).

File: src/winetricks [hunk 4]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+    # need native overrides to allow update and later checks to succeed
+    w_override_dlls native l3codeca.acm wmp wmplayer.exe wmvcore wmpnssci.dll
```
Comment (austin987) [inline src/winetricks]: Remove the .dll extension from wmpnssci.

File: src/winetricks [hunk 5]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+
+    # Crashes on exit, but otherwise ok; see https://bugs.winehq.org/show_bug.cgi?id=12633
```
Comment (austin987) [inline src/winetricks]: This comment can be removed, it's been fixed for years (or does wmp11 crash too?)

File: src/winetricks [hunk 6]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+    w_try_cd "$W_CACHE/$W_PACKAGE"
+    # https://docs.microsoft.com/en-us/windows/win32/wmp/redistributing-windows-media-player-11
+    "$WINE" wmp11-windowsxp-x64-enu.exe /q:A /c:"setup_wm.exe /Q /R:N"
```
Comment (austin987) [inline src/winetricks]: Add `w_try`.

This also unconditionally installs in quiet mode. Those flags should only be passed if W_OPT_UNATTENDED is set.

File: src/winetricks [hunk 7]
```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
...
+    #w_try cabextract -d "$W_TMP" ./wmp11-windowsxp-x64-enu.exe
```
Comment (austin987) [inline src/winetricks]: Please remove the commented out code.

File: src/winetricks (hunk 6 above)
Comment (soredake) [reply src/winetricks]: >Add w_try.

Installer tries to install some incompatible things and fails with 126 code at end.

File: src/winetricks (hunk 1 above)
Comment (soredake) [reply src/winetricks]: Done.

File: src/winetricks (hunk 2 above)
Comment (soredake) [reply src/winetricks]: Done.

File: src/winetricks (hunk 3 above)
Comment (soredake) [reply src/winetricks]: Done.

File: src/winetricks (hunk 4 above)
Comment (soredake) [reply src/winetricks]: Done.

File: src/winetricks (hunk 5 above)
Comment (soredake) [reply src/winetricks]: Done.

File: src/winetricks (hunk 7 above)
Comment (soredake) [reply src/winetricks]: Done.

**Comment by soredake** [comment]
Updated verb.

**Comment by soredake** [comment]
@basxto for now installation works only with silent mode, dunno how to make work in non-silent.

**Comment by luca-s** [comment]
I've tested it on Proton 5.0-3 with RE Revelations 2, works great thanks!

File: src/winetricks (hunk 6 above)
Comment (austin987) [reply src/winetricks]: Ok. Please add a comment.

**Comment by soredake** [comment]
Updated.

**Comment by jdetychey** [comment]
using the last winetricks isn't making the cut for RE6 on my side
https://github.com/ValveSoftware/Proton/issues/1252#issuecomment-622959164

**Pull request merged**